src/screens/SendAmountInput.js

```diff
@@ -118,6 +119,10 @@ class SendAmountInput extends React.Component {
       return ngettext(msgid`${amountAndToken} available`, `${amountAndToken} available`, available);
     };
 
+    const renderGhostElement = () => (
```

Comment:
Shouldn't we create a component `GhostElement`? Or is it too specific for each screen?

Reply:
I just used it in two different screens. They're specific in the sense that each needs a particular size. In the case where they were used, both have the same size. I'm not sure to be honest if we need a component for such a simple element, since it's only used twice so far.

Reply:
I agree. They seem very specific to each screen.